app/services/bus_logic/restricted_bus.py

```
@@ -0,0 +1,947 @@
+            return None
+
+    def get_station_name_by_ars_id(self, ars_id):
+        """ARS ID로 정류소명 조회"""
+        if not ars_id or len(ars_id) != 5:
+            return None
+            
+        try:
+            url = 'http://ws.bus.go.kr/api/rest/stationinfo/getStationByUid'
+            params = {'serviceKey': self.service_key, 'arsId': ars_id}
+            
+            response = requests.get(url, params=params, timeout=5, verify=False)
```

Ruff (0.15.1)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)